CHANGELOG.md

```diff
@@ -5,7 +5,7 @@
 See the [New Clojure project quickstart](https://blog.michielborkent.nl/new-clojure-project-quickstart.html) blog post for a gentle introduction into `neil`.
 
 ## 0.3.68
-
+- [#245](https://github.com/babashka/neil/issues/245): neil.el - neil-executable-path now can be set to `clj -M:neil`
```

Comment:
0.3.68 is already release, but you can add this above in a new section called `## Unreleased` :)

Reply:
This is elisp portion of it though. No straight version relation, no? I mean you don't cut a new version whenever elisp code changes? Or maybe it's the best to keep them both semvers synced - neil's and elisp package's?

Reply:
Note that a Github release also contains all the source files as a zip file and the elisp file at the tag 0.3.68 doesn't contain any changes, so we shouldn't list it under that release.

Reply:
Alright, I'll add it then under `##Unreleased`. But my question about the package version [here](https://github.com/agzam/neil/blob/neil-executable-path-compound-command/neil.el#L9), should I change that too?

Reply:
We don't have to synchronize that per se I guess, but most people will use the master version anyway I presume?

Reply:
We can of course bump that version at some point

Reply:
I bumped the package version. Will try to keep it aligned with "the last known compatible version" of the tool itself.